### PR TITLE
add getWorkflowExecution, updateWorkflowExecution, getTransitionExecution

### LINKED ISSUE #75: GET TransitionExecution
`client.makeGetRequest('/transitions/:transitionId/executions/:executionId')`

### LINKED ISSUE #91: PATCH WorkflowExecution

### LINKED ISSUE #92: GET WorkflowExecution

### PR DESCRIPTION
**Pull request by hexjelly**
Fixes #91, fixes #92, fixes #75

### CODE REVIEW

File: packages/las-sdk-core/src/client.spec.ts [hunk 1]
```diff
@@ -203,6 +204,17 @@ describe('Transitions', () => {
...
   });
 
+  describe('getTransitionExecution', () => {
+    test('valid request', async () => {
+      const testTransitionId = transitionId();
+      const testTransitionExecutionId = transitionExecutionId();
+      const getTransitionExecutionPromise = client.getTransitionExecution(testTransitionId, testTransitionExecutionId);
+      await expect(getTransitionExecutionPromise).resolves.toHaveProperty('executionId');
+      await expect(getTransitionExecutionPromise).resolves.toHaveProperty('status');
+      await expect(getTransitionExecutionPromise).resolves.toHaveProperty('transitionId');
```
Comment (augustak) [inline packages/las-sdk-core/src/client.spec.ts:214]: Missing some properties here: input, completedBy, startTime, endTime, logId

File: packages/las-sdk-core/src/client.spec.ts [hunk 2]
```diff
@@ -315,6 +327,42 @@ describe('Workflows', () => {
     });
   });
 
+  describe('getWorkflowExecution', () => {
+    test('valid request', async () => {
+      const testWorkflowId = workflowId();
+      const testWorkflowExecutionId = workflowExecutionId();
+      const getWorkflowExecutionPromise = client.getWorkflowExecution(testWorkflowId, testWorkflowExecutionId);
+      await expect(getWorkflowExecutionPromise).resolves.toHaveProperty('executionId');
```
Comment (augustak) [inline packages/las-sdk-core/src/client.spec.ts:335]: Missing logId

File: packages/las-sdk-core/src/client.spec.ts [hunk 3]
```diff
@@ -315,6 +327,42 @@ describe('Workflows', () => {
...
+  });
+
+  describe('updateWorkflowExecution', () => {
+    test('valid request', async () => {
+      const testWorkflowId = workflowId();
+      const testWorkflowExecutionId = workflowExecutionId();
+      const nextTransitionId = transitionId();
+      const data = { nextTransitionId }
+      const getWorkflowExecutionPromise = client.updateWorkflowExecution(testWorkflowId, testWorkflowExecutionId, data);
+      await expect(getWorkflowExecutionPromise).resolves.toHaveProperty('executionId');
```
Comment (augustak) [inline packages/las-sdk-core/src/client.spec.ts:354]: Missing logId

File: packages/las-sdk-core/src/client.ts [hunk 4]
```diff
@@ -412,6 +424,35 @@ export class Client {
     return this.makeGetRequest<WorkflowExecutionList>(`/workflows/${workflowId}/executions`, options);
   }
 
+  /**
+   * Get a workflow execution, calls the GET /workflows/{workflow_id}/executions/{execution_id} endpoint.
```
Comment (augustak) [inline packages/las-sdk-core/src/client.ts:428]: use camelCase

File: packages/las-sdk-core/src/client.ts [hunk 5]
```diff
@@ -412,6 +424,35 @@ export class Client {
...
+   * @param executionId Id of the execution to get
+   * @returns Workflow execution response from REST API
+   */
+  async getWorkflowExecution(workflowId: string, executionId: string): Promise<WorkflowExecution> {
+    return this.makeGetRequest(`/workflows/${workflowId}/executions/${executionId}`);
+  }
+
+  /**
+   * Retry or end the processing of a workflow execution,
+   * calls the PATCH /workflows/{workflow_id}/executions/{execution_id} endpoint.
```
Comment (augustak) [inline packages/las-sdk-core/src/client.ts:440]: use camelCase

File: packages/las-sdk-core/src/client.ts [hunk 6]
```diff
@@ -538,11 +579,12 @@ export class Client {
    * @param deleteDocuments Set to true to delete documents in batch before deleting batch
    * @returns Batch response from REST API
    */
-  async deleteBatch(batchId: string, deleteDocuments: boolean = false): Promise<Batch> {
+  async deleteBatch(batchId: string, deleteDocuments = false): Promise<Batch> {
```
Comment (augustak) [inline packages/las-sdk-core/src/client.ts:582]: Don't need type for boolean?

**Review by augustak — CHANGES_REQUESTED** [review]

File: packages/las-sdk-core/src/client.ts (hunk 6 above)
Comment (hexjelly) [reply packages/las-sdk-core/src/client.ts:582]: By setting a default value, the type is inferred

**Review by augustak — APPROVED** [review]

**Pull request merged**